packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -1028,27 +1032,28 @@ class DefaultTextStyleTransition extends AnimatedWidget {
 /// ** See code in examples/api/lib/widgets/transitions/animated_builder.0.dart **
 /// {@end-tool}
 ///
-/// {@template flutter.flutter.animatedbuilder_changenotifier.rebuild}
-/// ## Improve rebuilds performance using AnimatedBuilder
+/// ## Improve rebuilds performance using [AnimatedBuilder]
 ///
-/// Despite the name, [AnimatedBuilder] is not limited to [Animation]s. Any subtype
-/// of [Listenable] (such as [ChangeNotifier] and [ValueNotifier]) can be used with
-/// an [AnimatedBuilder] to rebuild only certain parts of a widget when the
-/// [Listenable] notifies its listeners. This technique is a performance improvement
-/// that allows rebuilding only specific widgets leaving others untouched.
+/// Despite the name, [AnimatedBuilder] is not limited to [Animation]s. Any
+/// subtype of [Listenable] (such as [ChangeNotifier] and [ValueNotifier]) can
+/// be used with an [AnimatedBuilder] to rebuild only certain parts of a widget
+/// when the [Listenable] notifies its listeners. This technique is a
+/// performance improvement that allows rebuilding only specific widgets leaving
+/// others untouched.
```

Comment:
Should we somehow mention ListenableBuilder right here? (with the same information that it is the same, but has a better name)